### ISSUE TITLE
Missing howto.txt

### ISSUE THREAD

**Issue by Perlence** [rr-/drill#2]
I installed drill via pip and I get this error when I try to read how-to:

```
FileNotFoundError: [Errno 2] No such file or directory: '/home/sviatoslav/.local/venvs/drillsrs/lib/python3.6/site-packages/drillsrs/data/howto.txt'
```

You should probably add MANIFEST.in and check in `howto.txt` there.

**Comment by rr-** [comment]
Thanks for the report. Employing the tip at http://stackoverflow.com/a/13783919 seemed to work. I released 0.3 that contains the fix suggested there.
If it turns out that it doesn't work and one indeed needs `MANIFEST.in`, I'll create it, but I'd like to avoid creating any extra files in the root directory if possible.

**Comment by Perlence** [comment]
Thanks, it works now!
